be/src/common/config.cpp

```diff
@@ -1493,6 +1493,9 @@ DEFINE_mBool(enable_compaction_pause_on_high_memory, "true");
 
 DEFINE_mBool(enable_calc_delete_bitmap_between_segments_concurrently, "false");
 
+// the max length of segments key bounds, in bytes
+DEFINE_mInt32(segments_key_bounds_truncation_threshold, "-1");
```

Comment:
why not keep it a extremely large value to get rid of the check `segments_key_bounds_truncation_threshold > 0`